src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +43,38 @@
      * Андрей - 600
      * София - 900</p>
      */
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
             throws Exception {
-        return null;
+        StringBuilder str = new StringBuilder();
+        LocalDate fromDate = LocalDate.parse(dateFrom, formatter);
+        LocalDate toDate = LocalDate.parse(dateTo, formatter);
+        int workinHours;
```

Comment:
@Merkald try to declare variables as close as possible to the first usage

Reply:
@Merkald did you consider to fix this?

Reply:
In previous commit was declare variables as close as possible to the first usage, in this deleted "fromDate" and "toDate" variables.